FormalConjectures/ErdosProblems/17.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Erdős Problem 17
+*Reference:* [erdosproblems.com/17](https://www.erdosproblems.com/17)
+-/
+
```

Comment:
```suggestion
-/

open Filter

open scoped Asymptotics
```
For the suggestion about using `IsBigO` (and corresponding notation) we need to open this.